conn.go

```diff
@@ -0,0 +1,66 @@
+package cleisthenes
+
+import (
+	"sync"
+
+	"github.com/DE-labtory/cleisthenes/pb"
+)
+
+type innerMessage struct {
+	*pb.Message
+	OnErr     func(error)
+	OnSuccess func(interface{})
+}
+
+type Message struct {
+	*pb.Message
+	Conn Connection
+}
+
+type Connection interface {
+	Send(data []byte, typ string, successCallBack func(interface{}), errCallBack func(error))
+	GetIP() string
```

Comment:
In Go convention, Get is not used. How about `func IP()`?

Reply:
`Ip` plz